libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.scss

```diff
@@ -0,0 +1,3 @@
+.sky-checkbox-group-hint-text {
+  margin-bottom: var(--sky-margin-stacked-lg);
```

Comment:
Why not use the `sky-margin-stacked-lg` CSS class instead of creating a new one?

Reply:
fixed